ambry-store/src/main/java/com.github.ambry.store/StoreFindToken.java

```diff
@@ -52,23 +54,25 @@
   private final Offset offset;
   private final StoreKey storeKey;
   private final UUID sessionId;
+  private final UUID incarnationId;
   private long bytesRead;
 
-  StoreFindToken() {
-    this(Type.Uninitialized, null, null, null);
+  StoreFindToken(UUID incarnationId) {
```

Comment:
not sure why an `Uninitialized ` token needs to start with an incarnation ID. `Uninitialized ` implies that nothing is initialized (not that it does not have a session ID either).